lib/atscppapi/examples/websocket/WebSocket.cc

```diff
@@ -0,0 +1,246 @@
+#include "WebSocket.hh"
+
+#include <iostream>
+#include "ts/ink_base64.h"
+#include "openssl/evp.h"
+
+using namespace atscppapi;
+
+#define SAY(a) std::cout << a << std::endl;
```

Comment:
Quite a few atscppapi examples appear to print to stdout, and I feel that this is appropriate for demonstration code. If you wanted to turn this into a provided plugin type, then this and quite a few other changes would of course be necessary.